### ISSUE TITLE
Optimisation

### ISSUE THREAD

**Issue by sprince0031** [sprince0031/UL_Masters#86]
Error/loss function that best fits the problem set.

**Comment by sprince0031** [comment]
We used the cross-entropy loss considering it is a multi-class classification problem.
